modules/ui/note_comments.js

```diff
@@ -68,7 +68,12 @@ export function uiNoteComments() {
         mainEnter
             .append('div')
             .attr('class', 'comment-text')
-            .html(function(d) { return d.html; });
+            .html(function(d) { return d.html; })
+            .selectAll('a')
+                .filter(isExternalLink)
+                .attr('rel', 'noopener')
+                .attr('rel', 'nofollow')
```

Comment:
Setting attributes via `attr` in D3 will override the existing value, so if you want both values you'll have to do `                .attr('rel', 'noopener nofollow')` instead.